src/scripts/Field.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const Cell = require('./Cell');
+
+class Field {
+  constructor(element) {
+    this.cells = [];
+
+    const rows = element.querySelectorAll('.field-row');
+
+    for (let i = 0; i < rows.length; i++) {
```

Comment:
Consider using 'forEach' or 'map' instead of the 'for' loop for better readability and performance. For instance, you can refactor this part like 'rows.forEach((row, i) => {...})'.